userprofile/templates/userprofile/support.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}{% blocktrans %}{{ brand_name }} Support {% endblocktrans %} | {{ block.super }}{% endblock %}
+
+{% block content %}
+
+<div class="page-header">
+	<h1>{% blocktrans %}{{ brand_name }} Support {% endblocktrans %}</h1>
+</div>
+
+<div class="row">
+	<div class="col-xs-12">
+		<h2 >
+			{% blocktrans %}Students{% endblocktrans%}
+		</h2>
+		<p>{% trans "Students should contact course personnel for support." %}</p>
```

Comment:
I think we should include this small student section in the template `support_channels.html` so that the student section can also be easily customised with `local_templates` in each site and institution.  This `userprofile/support.html` is not supposed to be overridden. When you do that, it would also be easy to include the links that I gave you earlier (Aalto Into links to student services).